plugin/pulp/plugin/profiler.py

```diff
@@ -0,0 +1,32 @@
+class Profiler(object):
```

Comment:
Agreed this may go away before 3.0 is released. I think a lot of what is being built currently is still subject to change as we get into the plugin conversion process.